app/src/main/res/values/strings.xml

```diff
@@ -20,14 +20,13 @@
     <string name="mode_animation">Animation</string>
     <string name="mode_laser">Laser</string>
     <string name="send_button">SEND</string>
-    <string name="preview_button">PREVIEW</string>
     <string name="permission_required">Permission required</string>
-    <string name="txt_turn_on_bluetooth">Turn on Bluetooth and try again</string>
     <string name="no_device_found">No device found. Please try again.</string>
     <string name="select_drawable_text">Select Drawable:</string>
-    <string name="select_drawable">Please select Drawable to be previewed</string>
```

Comment:
Please don't remove these strings for now. Might be useful.

Reply:
Sir, we are using automatic preview now, the second one will definitely not be used. We can keep the first one though